hadoop-client-modules/hadoop-client-integration-tests/src/test/java/org/apache/hadoop/example/ITUseMiniCluster.java

```diff
@@ -73,13 +78,22 @@ public void clusterUp() throws IOException {
         .numDataNodes(3)
         .build();
     cluster.waitActive();
+
+    conf.set("yarn.scheduler.capacity.root.queues", "default");
+    conf.setInt("yarn.scheduler.capacity.root.default.capacity", 100);
+    yarnCluster = new MiniYARNCluster(getClass().getName(), 1, 1, 1, 1);
+    yarnCluster.init(conf);
+    yarnCluster.start();
   }
 
   @After
   public void clusterDown() {
     if (cluster != null) {
       cluster.close();
     }
+    if (yarnCluster != null) {
```

Comment:
all services are closeable, so use `IOUtil.cleanupWithLogger()` & let it handle null checks and exceptions

Reply:
Nice. Didn't know this util method.